api/src/main/kotlin/io/spine/protodata/renderer/SourceFile.kt

```diff
@@ -202,16 +239,24 @@ private constructor(
     /**
      * Obtains the entire content of this file.
      */
+    @Deprecated("Use `text()` instead.", ReplaceWith("text()"))
     public fun code(): String {
-        initializeCode()
-        return code
+        return text().value
     }
 
     /**
      * Obtains the entire content of this file as a list of lines.
      */
     public fun lines(): List<String> {
-        return lineSplitter.splitToList(code())
+        return text().splitLines()
```

Comment:
Why don't we have `lines` as a property of `Text`, which returns a list of `Line`s? And a `Line` can have a `number` property and all necessary operations we already have and may want to have in the very near future.

I'm not suggesting to do it right in this PR. But we definitely need more powerful abstractions.